src/components/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  findParentPerson: (parentName: string) => Person | undefined;
+};
+
+const getLinkClass = (person: Person) => {
+  return classNames({ 'has-text-danger': person.sex === 'f' });
+};
+
+export const PersonLink: React.FC<Props> = ({ person, findParentPerson }) => {
+  const { slug } = useParams();
+  const location = useLocation();
+
+  const renderParentLink = (parentName: string | null) => {
+    const parent = parentName ? findParentPerson(parentName) : undefined;
+
+    return parent ? (
+      <Link
+        className={getLinkClass(parent)}
+        to={`/people/${parent.slug}${location.search}`}
+      >
+        {parent.name}
+      </Link>
+    ) : (
+      parentName || '-'
+    );
+  };
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === person.slug,
+      })}
```

Comment:
You are using `classNames` function to conditionally add the 'has-background-warning' class. This is a good approach, but since you are only checking for one condition, you could simplify this by using a ternary operator directly in the className property. For example: `className={slug === person.slug ? 'has-background-warning' : ''}`